mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -676,8 +691,20 @@ function SourceCellRenderer(props) {
 SourceCellRenderer.propTypes = { value: PropTypes.object };
 
 function VersionCellRenderer(props) {
-  const { version, name, type } = props.value;
-  return Utils.renderSourceVersion(version, name, type) || EMPTY_CELL_PLACEHOLDER;
+  // prettier-ignore
```

Comment:
This seems weird. Why are we ignoring prettier? cc @mohamad-arabi

Reply:
This function looks like in our private repo:

```js
function VersionCellRenderer(props) {
  // prettier-ignore
  const {
    // BEGIN-EDGE
    someProp,
    // END-EDGE
    version,
    name,
    type,
  } = props.value;
  return (
    // prettier-ignore
    Utils.renderSourceVersion(
      // BEGIN-EDGE
      someProp,
      // END-EDGE
      version,
      name,
      type,
    ) || EMPTY_CELL_PLACEHOLDER
  );
}
```

Reply:
We might want to add a comment above `prettier-ignore` for clarity.